### DISCUSSION TITLE
System freezes after connecting

### DISCUSSION THREAD

**Discussion by stevenmccormack** [rustdesk/rustdesk#8723] [General]
### Bug Description

After upgrading from 1.2.3 to 1.2.6 I have encountered some cases where a connection to a remote PC froze my system.
I can connect to the client with the password prompt showing, when the remote user  accepts the connection my screens turn black for a second. Then my Desktop reappears but I am no longer able to do anything on the system. The mouse pointer is stuck and the keyboard cannot be used as well (pressing num lock will no longer trigger the led on the keyboard ).
 When this happens all I can do is hold the power cycle my PC. 

### How to Reproduce

I am unable to force the bug. It does never happen directly after a reboot but I am unaware on how to enforce it.

### Expected Behavior

RustDesk and my PC work normally 

### Operating system(s) on local side and remote side

Linux

### RustDesk Version(s) on local side and remote side

1.2.6

### Screenshots

As the computer freezes, I am unable prodice screenshots

### Additional Context

Please find below my system details:
OS: Debian GNU/Linux trixie trixie/sid x86_64
Kernel: Linux 6.9.8-amd64
Shell: zsh 5.9
Display (LG ULTRAGEAR): 2560x1440 @ 165Hz *
Display (LG ULTRAGEAR): 2560x1440 @ 165Hz
Display (DELL U2715H): 2560x1440 @ 60Hz
DE: KDE Plasma 5.27.11
WM: KWin (X11)
CPU: AMD Ryzen 9 7900X3D (24) @ 5,66 GHz
GPU: AMD Radeon RX 7900 XTX @ 0,02 GHz [Discrete]
Memory: 4,56 GiB / 62,48 GiB (7%)

**Comment by bh1san** [comment]
Does anyone have a solution for this? My issue is the opposite - after connecting, the remote PC freezes.

**Reply by rustdesk** [reply]
Please paste your log https://github.com/rustdesk/rustdesk/wiki/FAQ#access-logs (remote side)
